### PR TITLE
0530 prepare for 5.1.0 alpha.1

### PR DESCRIPTION
**Pull request by zmstone**
* Update `release-50` to `release-51` in release scripts.
* Finalize `master.bpapi` as `5.0.bpapi`

### CODE REVIEW

File: apps/emqx/include/emqx_release.hrl [hunk 1]
```diff
@@ -32,10 +32,10 @@
 %% `apps/emqx/src/bpapi/README.md'
 
 %% Community edition
--define(EMQX_RELEASE_CE, "5.0.26").
+-define(EMQX_RELEASE_CE, "5.1.0-alpha.1").
```
Comment (id) [inline apps/emqx/include/emqx_release.hrl:35]: ?

File: scripts/rel/cut.sh [hunk 2]
```diff
@@ -270,7 +270,7 @@ else
         *beta*)
             true
             ;;
-        e5.0.0*)
+        e5.1.0*)
             # the first release has no change log
```
Comment (id) [inline scripts/rel/cut.sh:274]: We don't want to show changes between 5.0 and 5.1?

File: scripts/rel/cut.sh [hunk 3]
```diff
@@ -13,14 +13,14 @@ usage() {
...
-  e5.0.0-beta.6
+  v5.1.1
+  e5.1.0-beta.6
 
 options:
   -h|--help:         Print this usage.
 
   -b|--base:         Specify the current release base branch, can be one of
-                     release-50
+                     release-51
```
Comment (id) [inline scripts/rel/cut.sh:23]: May be we can just use `release` at this point

Comment (zmstone) [reply scripts/rel/cut.sh:23]: `release` is not quite search friendly. 
how about `release-candidate`?

File: apps/emqx/include/emqx_release.hrl (hunk 1 above)
Comment (zmstone) [reply apps/emqx/include/emqx_release.hrl:35]: yes, both will enter 5.1 from now

**Review by id — APPROVED** [review]

**Pull request merged**